lib/modules/manager/index.spec.ts

```diff
@@ -89,6 +90,22 @@ describe('modules/manager/index', () => {
       const mgr = mgrs.get(name)!;
       expect(validate(mgr, name)).toBeTrue();
     }
+
+    const urlRegex = regEx(/^\[.*\]\(.*\)$/);
+
+    for (const [managerName, managerModule] of Object.entries(loadedMgr)) {
+      const urls = (managerModule as { urls?: string[] }).urls ?? [];
```

Comment:
```suggestion
      const urls = (managerModule as { urls?: string[] })?.urls ?? [];
```
why the cast? 🤔 we've an interface for them, see docs generation inside tools folder

Reply:
Its not needed for the manager module because `ManagerApi` extends `ModuleApi` which has the preoperty `urls`.

But, the `VersioningApi` is stand-alone and does not extend the `ModuleApi`, so it doesn't allow the `urls` property. 
Needs to be fixed in diff PR, first.